### PR TITLE
Updates to match WG resolutions

### PR DESCRIPTION
**Pull request by csuwildcat**

### CODE REVIEW

File: docs/spec/resolution.md [hunk 1]
```diff
@@ -75,22 +79,16 @@
...
-            2. Apply the patch as directed by the [Patch Action](#standard-patch-actions) type specified by the `action` property. If any part of the patch fails or produces an error, reverse all modifications to the DID's state and exit [update operation processing](#update-operation-processing) and advance to [compiled state processing](#compiled-state-processing).
+            1. Validate the entry in accordance any requirements imposed by the [Patch Action](#standard-patch-actions) type indicated by the `action` value of the entry. If the entry is valid, proceed, if the entry fails validation, skip the entry and iterate forward to the next operation.
+            2. Apply the patch as directed by the [Patch Action](#standard-patch-actions) type specified by the `action` property. If any part of the patch fails or produces an error, reverse all modifications to the DID's state and skip the entry and iterate forward to the next entry.
```
Comment (sandrask) [inline docs/spec/resolution.md:91]: Should it be "If any part of the patch fails or produces an error, reverse all modifications to the DID's state, exit iterating patches and iterate forward to the next operation."

File: docs/spec/parameters.md [hunk 2]
```diff
@@ -20,5 +20,6 @@ Each version of the protocol will define a set of protocol rules and parameters
 | `MAX_PROOF_FILE_SIZE` { #max-proof-file-size }       | Maximum compressed map file size.                        |               2.5 MB |
 | `MAX_CHUNK_FILE_SIZE`  { #max-chunk-file-size }  | Maximum compressed chunk file size.                      |              10 MB |
 | `MAX_ENCODED_HASH_LENGTH`   | Maximum accepted string length of an encoded hash.                            |          100 bytes |
-| `MAX_OPERATION_SIZE`        | Maximum uncompressed operation size.                                          |               1 kb |
-| `MAX_OPERATION_COUNT`       | Maximum number of operations per batch.                                       |             10,000 |
\ No newline at end of file
+| `MAX_DELTA_SIZE` { #max-delta-size }        | Maximum uncompressed operation delta size.                                          |               1 kb |
+| `MAX_OPERATION_COUNT`       | Maximum number of operations per batch.                                       |             10,000 |
+| `MAX_OPERATION_HASH_LENGTH` { #max-operation-hash-length }       | Maximum number of operations per batch.                                       |             50 bytes |
```
Comment (sandrask) [inline docs/spec/parameters.md:25]: Description should be changed for MAX_OPERATION_HASH_LENGTH

File: docs/spec/parameters.md [hunk 3]
```diff
@@ -20,5 +20,6 @@ Each version of the protocol will define a set of protocol rules and parameters
 | `MAX_PROOF_FILE_SIZE` { #max-proof-file-size }       | Maximum compressed map file size.                        |               2.5 MB |
 | `MAX_CHUNK_FILE_SIZE`  { #max-chunk-file-size }  | Maximum compressed chunk file size.                      |              10 MB |
 | `MAX_ENCODED_HASH_LENGTH`   | Maximum accepted string length of an encoded hash.                            |          100 bytes |
-| `MAX_OPERATION_SIZE`        | Maximum uncompressed operation size.                                          |               1 kb |
-| `MAX_OPERATION_COUNT`       | Maximum number of operations per batch.                                       |             10,000 |
\ No newline at end of file
+| `MAX_DELTA_SIZE` { #max-delta-size }        | Maximum uncompressed operation delta size.                                          |               1 kb |
```
[the remark this answers is not in the document]
Comment (sandrask) [reply docs/spec/parameters.md:23]: Also can you change default compression algorithm from "ZIP" to "GZIP"

File: docs/spec/parameters.md (hunk 2 above)
Comment (csuwildcat) [reply docs/spec/parameters.md:25]: Good catch, will fix now

File: docs/spec/resolution.md (hunk 1 above)
Comment (csuwildcat) [reply docs/spec/resolution.md:91]: Good catch, will fix all of these paste omissions.

File: docs/spec/parameters.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sandrask) [reply docs/spec/parameters.md:23]: Are you planning to add MAX_PROOF_SIZE in this PR?

[the remark this answers is not in the document]
Comment (csuwildcat) [reply docs/spec/parameters.md]: @thehenrytsai @isaacJChen I think we were going to add a line that proofs must validate according to the implementation schema, which could include size checks for values.

[the remark this answers is not in the document]
Comment (isaacJChen) [reply docs/spec/parameters.md]: @sandrask I made a comment about max proof side not needed and the reason. Basically we are doing a strict schema check which doesn't allow the size to vary. Let me know if you disagree.

https://github.com/decentralized-identity/sidetree/issues/950

**Review by isaacJChen — APPROVED** [review]

File: docs/spec/parameters.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sandrask) [reply docs/spec/parameters.md]: @isaacJChen thanks for the explanation

**Pull request merged**